sopel/modules/currency.py

```diff
@@ -1,102 +1,189 @@
 # coding=utf-8
 # Copyright 2013 Elsie Powell, embolalia.com
+# Copyright 2019 Mikkel Jeppesen
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+import time
```

Comment:
Should be below `re`. (See below comment.)

Reply:
Should've fixed this.